recipes/r-saccharis/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+r_base:
+  - 4.3.*
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)